students/nDruP/lesson09/donor.py

```diff
@@ -0,0 +1,54 @@
+class Donor():
+    def __init__(self, title, gift):
+        self._name = title
+        self._history = []
+        if type(gift) == int or type(gift) == float:
+            self._history.append(round(gift, 2))
+        elif type(gift) == list:
+            for x in gift:
+                self._history.append(round(x, 2))
+
+    def __len__(self):
+        return len(self._history)
+    
+    @property
+    def name(self):
+        return self._name
+
+    @property
+    def history(self):
+        return self._history
```

Comment:
You don't need to use a `@property` for these attributes since you aren't doing any extra work beyond defining them it's more appropriate to set them directly in the constructor